docs/api/vega-wallet/v2-api/integrate-wallet-service.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Integrate the wallet service
+hide_title: false
+sidebar_position: 5 
+---
+
+This guide will walk you through how to set up the Vega  Wallet from scratch to use when building a Vega dApp, or integrating with CI/CD. 
+
+You can find the current working CI/CD implementation in the [front-end GitHub repo ↗](https://github.com/vegaprotocol/frontend-monorepo/blob/develop/.github/actions/setup-vegawallet/action.yml).
+
+Depending on your end-goal, you may be able to skip some steps. API token-related steps are only needed for end-to-end and CI/CD integration. Also, if you already have Vega Wallet prepared and configured, you may only need the last step to run the wallet.
+
+1. **Create two files for your `passphrase` and `recovery` phrase** - `passphrase` will be used as a wallet and API token password (you may want to use two different ones), and `recovery` will contain recovery phrase of the wallet you want to use (you can skip it if you don’t have a wallet yet, and want to create a new one).
+
+2. **Download and install the Vega Wallet software.** 
+    There are two ways you can do that:
+    a. Download the latest release of the [Vega software ↗](https://github.com/vegaprotocol/vega/releases) and unzip it to a shared location. To test if it worked, run the `vega version` command.
```

Comment:
The command is now 

Vega software check

Reply:
@ValentinTrinque did something change? As for `v0.67.3` there are two commands available
- `vega version` - it returns version of vega binaries (SHOULD be the same as vegawallet binaries version)
- `vega wallet software version` - it returns vegawallet binaries version

There is no such command as `vega software check`.

Reply:
Sorry I meant:

vega **wallet** software check